src/controllers/neighborhoods.ts

```diff
@@ -14,4 +14,12 @@ neighborhoodsRouter.get('/', catchError(async (_req, res) => {
   }
 }));
 
+neighborhoodsRouter.delete('/:id', catchError(async (req, res) => {
+  const deletedNeighborhood = await prisma.neighborhood.delete({
+    where: { id: +req.params.id },
```

Comment:
What does '+' mean? I don't think I've encountered this before.

Reply:
It coerces `req.params.id` to a Number

Reply:
I'd say that this is why `Number(req.params.id)` is often preferred, since it's a bit more self-documenting.

Reply:
Now, I should point out that this function will also cascade delete all related records in other tables: all rows in the `neighborhoods_users` table, all rows in the `requests` table, and then all rows in the `responses` table that respond to a deleted request. I'm going to run a few tests of my own to make sure these work correctly.

Reply:
Ok. I'm finding that all the cascade deletes work properly (I inserted a neighborhood, a request and a response, deleted the neighborhood, and the request and response also deleted as expected). However, there are now other issues around the `admin_id` field that I've uncovered. I'll see if I can put up an issue for that.

Reply:
Yes, but we should make a separate issue after we implement the login functionality.